shared/src/packages/comments/libs/types/comment-entity.type.ts

```diff
@@ -0,0 +1,9 @@
+type CommentEntityType = {
+  id: number;
+  text: string;
+  userId: number;
+  articleId: number;
+  publishedAt: string | null;
```

Comment:
Or leave it with `publishedAt` field but map createdAt column value to it
```suggestion
type CommentEntity = {
  id: number;
  text: string;
  userId: number;
  articleId: number;
  createdAt: string;
```